Products/PleiadesEntity/browser/attestations.py

```diff
@@ -91,6 +91,18 @@ def __call__(self):
         return u'<ul class="placeChildren">' + u'\n'.join(rows) + '</ul>'
 
 
+class RepresentativePoint(BrowserView):
+    """representative point data
+    """
+
+    def __call__(self):
+        # GeoJson stores longitude first, followed by latitude
+        # This view returns latitude, longitude
+        point = str(representative_point(self.context)['coords'][1]) + ', ' +\
+            str(representative_point(self.context)['coords'][0])
+        return '<br/>' + point
```

Comment:
@witekdev it's possible for `representative_point` to return `None`, so we should handle that here. Also it takes some work to calculate the representative point, so we should avoid calling it twice. I'll fix it.